From 88ade216df46b89f57f1fdc26345b663f4009acd Mon Sep 17 00:00:00 2001
From: Michael Pauzner <mpauzner@habana.ai>
Date: Tue, 01 Apr 2025 16:35:49 +0300
Subject: [PATCH] [SW-221274] g3: add serdes config region

Add SerDes configuration partition to SPI flash.

Change-Id: I4ead240ff019c801365fbda104ff99aab967bf7b
Signed-off-by: Michael Pauzner <mpauzner@habana.ai>
(cherry picked from commit ceadd170fe718b45376ef89187bc98b86903484f)
---

diff --git a/gaudi3/gaudi3_mem_addr_dt.h b/gaudi3/gaudi3_mem_addr_dt.h
index 243eaa5..e8b2a20 100644
--- a/gaudi3/gaudi3_mem_addr_dt.h
+++ b/gaudi3/gaudi3_mem_addr_dt.h
@@ -317,8 +317,8 @@
 #define RMA_2_PARTITION_OFFSET				190000
 #define RMA_2_PARTITION_SIZE				10000
 
-#define SPI_FLASH_HL325_LFS_OFFSET			1c0000
-#define SPI_FLASH_HL325_LFS_SIZE			40000
+#define SPI_SERDES_CONFIG_OFFSET			1c0000
+#define SPI_SERDES_CONFIG_SIZE				40000
 #define SPI_FLASH_HL338_LFS_OFFSET			c00000
 #define SPI_FLASH_HL338_LFS_SIZE			400000
 
